src/test/java/steps/StepsDefinition.java

```diff
@@ -0,0 +1,40 @@
+package steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import pages.MenuPage;
+import pages.WebViewPage;
+
+public class StepsDefinition {
+
+    @Given("^I am in the WebDriverIo app$")
```

Comment:
Try to always make actions in **Given** and **When**, and assertions in **Then** blocks.
When('I press X button') --> Just the action
Then('This happens') --> The assertions of the previous actions

Reply:
As the application starts before the given, this test starts it validating that it is in the application. I will take into account the recommendation to rethink the following tests.